tests/conftest.py

```diff
@@ -82,10 +82,11 @@ def needs_credentials(
     check_user_kerberos_creds,
 ):
     monkeypatch.setenv("GROUP", TestUnit.test_group)
-    # yield creds.get_creds({"role": "Analysis"})
+    yield creds.get_creds({"role": "Analysis"})
     cred_set_token = needs_token
     cred_set_proxy = needs_x509_user_proxy
     yield creds.CredentialSet(token=cred_set_token.token, proxy=cred_set_proxy.proxy)
+    yield creds.CredentialSet(token=cred_set_token.token)
```

Comment:
It looks like we're yielding three times here, which appears to me to be a typo.  Did you mean to do that, or should this just be these three lines in this fixture?

```python
    cred_set_token = needs_token
    cred_set_proxy = needs_x509_user_proxy
    yield creds.CredentialSet(token=cred_set_token.token, proxy=cred_set_proxy.proxy)
```